<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>四象限光斑位置处理软件 （@KYPhotonics）</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/icons/images/KY-Logo-3232.ico</normaloff>:/icons/images/KY-Logo-3232.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0">
      <item>
       <widget class="QCustomPlot" name="rawDataPlot" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1169</width>
          <height>212</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCustomPlot" name="sumDataPlot" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
        <item>
         <widget class="QCustomPlot" name="fourQuadPlot" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>128</width>
            <height>78</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>XDIFF</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="xdiffDisplay">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>YDIFF</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="ydiffDisplay"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Sum</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="sumDisplay">
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>存储设置</string>
          </property>
          <widget class="QWidget" name="">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>301</width>
             <height>151</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLineEdit" name="saveDirPathLineEdit">
              <property name="placeholderText">
               <string>C:\</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="saveCheckbox">
              <property name="text">
               <string>是否保存数据</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="chooseDirPathBtn">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>选择</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>采集设置</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>321</width>
             <height>151</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <item row="1" column="0" colspan="2">
             <widget class="QPushButton" name="startBtn">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>开始</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="sampleRateSampleBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="0" column="2" colspan="2">
             <widget class="QPushButton" name="initBtn">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>连接</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" colspan="2">
             <widget class="QPushButton" name="stopBtn">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>停止</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>采样率</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>stopBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1178</x>
     <y>739</y>
    </hint>
    <hint type="destinationlabel">
     <x>1261</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>initBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>initBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1178</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>1307</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1019</x>
     <y>739</y>
    </hint>
    <hint type="destinationlabel">
     <x>1248</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sampleRateSampleBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sampleRateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1019</x>
     <y>673</y>
    </hint>
    <hint type="destinationlabel">
     <x>1250</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>saveDataCheckBoxChecked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>667</y>
    </hint>
    <hint type="destinationlabel">
     <x>1249</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooseDirPathBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>chooseDirectoryBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>729</y>
    </hint>
    <hint type="destinationlabel">
     <x>1233</x>
     <y>612</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>stopBtnClicked()</slot>
  <slot>initBtnClicked()</slot>
  <slot>startBtnClicked()</slot>
  <slot>sampleRateChanged()</slot>
  <slot>saveDataCheckBoxChecked()</slot>
  <slot>chooseDirectoryBtnClicked()</slot>
 </slots>
</ui>
